frontend/src/components/modal/user/UserInfoModal.vue

```diff
@@ -263,6 +260,10 @@ export default {
         this.editingEnabled = true
       }
     },
+    safeClose() {
+      this.userData = { ...this.initialData }
```

Comment:
From my testing it looks like this (along with the `initialData` field) isn't required, since userData is a deep copy

Reply:
Confirmed unneeded now due to the userData deep copy resolution. Reverting.